modules/raft/src/main/java/org/apache/ignite/internal/raft/server/impl/JraftServerImpl.java

```diff
@@ -457,6 +457,14 @@ public boolean startRaftNode(
         }
     }
 
+    @Override
+    public CompletableFuture<Long> raftNodeReadyFuture(ReplicationGroupId groupId) {
+        RaftGroupService jraftNode = nodes.entrySet().stream().filter(entry -> entry.getKey().groupId().equals(groupId))
```

Comment:
Are you sure that there will only one node with given groupId. Seems that it should be true for AI, but not for GG because of learners. Could you please create ticket for GG if it's possible to have multiple nodes there?